adapters/InMobi/InMobiAdapter/GADMediationAdapterInMobi.h

```diff
@@ -11,21 +11,24 @@
 // WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 // See the License for the specific language governing permissions and
 // limitations under the License.
+//
 
 #import <Foundation/Foundation.h>
 #import <GoogleMobileAds/GoogleMobileAds.h>
 #import <InMobiSDK/InMobiSDK.h>
 
 typedef NS_ENUM(NSInteger, GADMAdapterInMobiErrorCode) {
-  /// Missing server parameters.
-  GADMAdapterInMobiErrorInvalidServerParameters = 101,
-  /// Banner Size Mismatch.
-  GADMAdapterInMobiErrorBannerSizeMismatch = 102,
-  /// An ad is already loaded for this network configuration.
-  GADMAdapterInMobiErrorAdAlreadyLoaded = 103,
-  /// InMobi native ad returned with missing native assets, or required image assets failed to
-  /// download.
-  GADMAdapterInMobiErrorMissingNativeAssets = 104
+    /// Missing server parameters.
+    GADMAdapterInMobiErrorInvalidServerParameters = 101,
+    /// Banner Size Mismatch.
+    GADMAdapterInMobiErrorBannerSizeMismatch = 102,
+    /// An ad is already loaded for this network configuration.
+    GADMAdapterInMobiErrorAdAlreadyLoaded = 103,
```

Comment:
What happens if the publisher loads two InMobi ads consecutively? Will InMobi SDK return a failure back when trying to load the second one?

For example:
```
var ad1 = GoogleInterstitial(adUnit: unit) // Loads an InMobi ad.
var ad2 = GoogleInterstitial(adUnit: unit) // Also loads an InMobi ad. Will this fail?

ad2.show() // Show the ad.
ad1.show() // Question: will this fail?
```

Reply:
The above will result into different instances being created for Inmobi ad object by the adapter.
Hence the load and show will not fail.

But only in case of rewarded ad, there are checks is apdater code, 
to not allow the same placement to be loaded again. Hence the second load and show will fail.